lacework_node.tf

```diff
@@ -48,7 +49,8 @@ resource "kubernetes_secret" "lacework_config" {
   }
 
   data = {
-    "config.json" = local.merged_node_config
+    "config.json"         = local.merged_node_config
+    "syscall_config.yaml" = local.node_syscall_config_data
```

Comment:
Could you add a new Terraform variable to toggle syscall enabled / disabled and gate the inclusion of the syscall config with this toggle?

Reply:
Done!